python/pp_ser.py

```diff
@@ -327,7 +328,8 @@ def __ser_register(self, args):
         datatypes = dict(integer=["'int'", 'ppser_intlength'], real=['ppser_realtype', 'ppser_reallength'])
 
         if dirs[1] not in datatypes:
-            self.__exit_error(directive = args[0], msg = 'Data type "{0}" not understood. Valid types are: {1}'.format(dirs[1], ', '.join('"' + d + '"' for d in datatypes.keys())))
+            self.__exit_error(directive=args[0], msg='Data type "{0}" not understood. Valid types are: {1}'.
+                              format(dirs[1], ', '.join('"' + d + '"' for d in datatypes.keys())))
```

Comment:
Does PEP8 specify that in these cases one can put a newline just after the dot? I always wonder where should I break long `.format` statements...